labs/lab-1-html-and-css/mobilestyle.css

```diff
@@ -0,0 +1,82 @@
+
```

Comment:
Pick an indentation size and stick with it, please! 2, 4 and 8 spaces are the most common indentation sizes to use -- but once you pick one, stick with it!